sdk/src/main/java/com/mapbox/maps/plugin/MapPluginRegistry.kt

```diff
@@ -63,16 +65,16 @@ MapPluginRegistry(private val mapDelegateProvider: MapDelegateProvider) {
       if (instance is ViewPlugin) {
         val pluginView = instance.bind(
           mapView!!,
-          mapboxMapOptions.attrs,
-          mapboxMapOptions.pixelRatio
+          attrs,
```

Comment:
one question: the attrs was a part of the MapboxMapOptions, would it make sense to move it to the MapInitOptions? I also do see some benefits to separate these.

Reply:
Yeah, that makes sense.